src/pages/[platform]/build-a-backend/push-notifications/push-notifications-migration-guide/index.mdx

```diff
@@ -0,0 +1,160 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Migrate from v5 to v6',
+  description: '',
+  platforms: ['react-native', 'flutter', 'swift', 'android']
```

Comment:
Ditto with Swift & Android

Reply:
TY. Only `react-native` should stay given there is migration needed only there.

Reply:
Updated it to just `['react-native']`